### ISSUE TITLE
[jcomm] 

### ISSUE THREAD

**Issue by MartijnMaandag** [joomla/joomla-websites#1465]
Its me again ;-)
I try to have the information at the Dutch JUGs up to date. 
When I use the contact button nothing happens. Example: https://community.joomla.org/user-groups/europe/netherlands/1003-joomla-user-group-arnhem-nijmegen.html#
Firefox: 
No 'Name' and 'Email address'
Chrome and Edge: 'Name' and 'Email address' yes, but 'Send' has no effect.

**Comment by particthistle** [comment]
![image](https://user-images.githubusercontent.com/5515866/85947020-9f27cd80-b98b-11ea-8abf-3d182888263c.png)

**Confirming issue** - contact form in Windows Firefox has name and email address on my form, but the send does not do anything.

Tried it also with contacting my User Group https://community.joomla.org/user-groups/oceania/australia/3426-joomla-user-group-melbourne.html#

**Comment by conconnl** [comment]
@MartijnMaandag as discussed a few weeks ago, I contacted the developer.
They have been able to provided a solution and I just tested it.
At least the form reacts again when I try to use it.

**Comment by MartijnMaandag** [comment]
In Firefox; Still no 'Name' and 'Email address'

Chrome and Edge: 
Really sorry, but the message cannot be send. Please inform the site administrator or try again later.

**Comment by conconnl** [comment]
The name display depends on if you are logged in or not.
Have you emptied your browser cache? (personally i always have cache disabled)

I just tested Firefox, Chrome, Safari and Edge.
They all work for me. @particthistle can you also test?

Firefox:
**Not logged in:**
<img width="1419" alt="image" src="https://user-images.githubusercontent.com/11913918/87412048-cfe14700-c5c8-11ea-83ca-351dddd16dd6.png">

**Logged in:**
<img width="1407" alt="image" src="https://user-images.githubusercontent.com/11913918/87412106-e38cad80-c5c8-11ea-8d4b-f222f35a34f6.png">

**Comment by particthistle** [comment]
- Cleared cache
- Tested form in firefox
- Fields worked
- Form didn't submit: Really sorry, but the message cannot be send. Please inform the site administrator or try again later.

![Formtest](https://user-images.githubusercontent.com/5515866/87641595-ade3f200-c78b-11ea-902c-ccb5375b35e9.gif)

**Comment by conconnl** [comment]
To bad, the only thing I can do is send a message again to the developer.
Specifically if they do not replace the Sender value in generating mails.
Because you are in 9 out of 10 times not allowed to send mail out of the name of other people, that is considered spam.
I will drop a new message when I hear something.

At least it does already more then before.

**Comment by brianteeman** [comment]
Tested and confirming

1. Tested submitting the form with adblock **enabled** in my chrome browser on windows.
Form will not send - nothing happens when you click on the send button

2. Tested submitting the form with adblock **disabled** in my chrome browser on windows.
Form will not send - error message displayed (in bad engish) "Really sorry, but the message cannot be send. Please inform the site administrator or try again later."

If only joomla had a native contact form that could be used

**Comment by particthistle** [comment]
> Specifically if they do not replace the Sender value in generating mails.
> Because you are in 9 out of 10 times not allowed to send mail out of the name of other people, that is considered spam.

Agreed. Setting the from as a system joomla.org address (the site email?) and then making the reply-to set to the enquirers address is the solution.

**Comment by conconnl** [comment]
The developer confirmed.
At this moment there is no "reply_to" in the contact option part of the component, they will add it somewhere in the future, but do not know yet exactly.

The only option I have at the moment is changing the sender to our default and add the real sender info to the message.
The only thing that can happen; Someone replies and do not use the provided info and the reply won't go to the requestor.
@webdjam can you as DC tell me what you want to do?

**Comment by webdjam** [comment]
The contact form works fine on my side, when  I'm log in, @conconnl  can we implement another form like RsForm  ?

**Comment by conconnl** [comment]
The contact form is something related to the specific JUG. Creating it in RSForm, means connecting it to the other component. 

While the issue basically is the sender configuration. If we put in a fix sender, everybody would be able to use it. Only the receiver should make sure they reply to the real sender.
